src/aqua/installation-spell/src/aqua/deal_spell.aqua

```diff
@@ -60,15 +61,43 @@ func deal_install(ipfs: Multiaddr, worker_def_cid: CID, deal_id: Address):
         log(["Worker installation finished with status", status.str])
 
         if status.str == "install finished":
-            subnet_id, cre_errs <- createSubnetwork(deal_id)
-            log(["Subnet created", subnet_id])
-            -- subnet_id will be 'nil' if error happened on 'createSubnetwork'
+            subnet_id: ?string
+
+            now <- Peer.now_sec()
+            created <- Spell.get_u32("subnet_created")
+            registered <- Spell.get_u32("worker_registered")
+            if created.success:
+                if registered.success:
```

Comment:
`registered.success` and `created.success` will be always set to `true` (if sqlite works ok), but `.absent` value will differ

Reply:
and what's the problem?